diff -c -r DCE-Perl-0.14/ACL/ACL.xs DCE-Perl-0.14-sparc/ACL/ACL.xs
*** DCE-Perl-0.14/ACL/ACL.xs	Tue Nov 19 06:00:06 1996
--- DCE-Perl-0.14-sparc/ACL/ACL.xs	Wed Apr 23 11:45:52 1997
***************
*** 689,695 ****
  	set_hv = (HV*)SvRV(ST(1));
  
  	svp = hv_fetch(set_hv, "entry_type", 10, 1);
! 	if(SvTRUE(*svp)) 
  	    e->entry_info.entry_type = (sec_acl_entry_type_t)SvIV(*svp);
  
  	FETCH_SEC_ID(e->entry_info.tagged_union.id, set_hv, "id", 2);
--- 689,695 ----
  	set_hv = (HV*)SvRV(ST(1));
  
  	svp = hv_fetch(set_hv, "entry_type", 10, 1);
! 	if(SvOK(*svp))
  	    e->entry_info.entry_type = (sec_acl_entry_type_t)SvIV(*svp);
  
  	FETCH_SEC_ID(e->entry_info.tagged_union.id, set_hv, "id", 2);
